src/libappimage_shared/elf.c

```diff
@@ -117,41 +117,7 @@ static off_t read_elf64(FILE* fd)
 	return sht_end > last_section_end ? sht_end : last_section_end;
 }
 
-ssize_t appimage_get_elf_size(const char* fname) {
```

Comment:
Note for later: I'm pretty sure appimagetool uses this function, we might need to add an alias for the C++ stuff for it to continue to work.

Reply:
Fixed, included in the `appimage_legacy.h`. The newly written code should use `appimage_get_payload_offset`